enterprise/frontend/src/metabase-enterprise/sdk-plugins.js

```diff
@@ -0,0 +1,12 @@
+import { PLUGIN_IS_EE_BUILD } from "metabase/plugins";
+
+// SETTINGS OVERRIDES:
+PLUGIN_IS_EE_BUILD.isEEBuild = () => true;
+
+import "./shared";
+
+// SDK PLUGINS:
+
+import "./embedding";
+import "./embedding-sdk";
+import "./metabot";
```

Comment:
We discussed here https://metaboat.slack.com/archives/C063Q3F1HPF/p1746619027889539 that we can remove all except these.